template.php

```diff
@@ -1060,6 +1060,13 @@ function europa_preprocess_page(&$variables) {
   // Small fix to maxe the link to the start page use the alias with language.
   $variables['front_page'] = url('<front>');
 
+  $variables['settings'] = [
+    'header_home' => theme_get_setting('ec_europa_site_header_home', 'europa'),
```

Comment:
Now that I see with more attention the names of the variables, I have to say that a consistence doesn't exist in the choice of the same ones.